src/openforms/js/components/admin/form_design/Context.js

```diff
@@ -7,7 +7,7 @@ const FeatureFlagsContext = React.createContext({});
 FeatureFlagsContext.displayName = 'FeatureFlagsContext';
 
 const FormContext = React.createContext({
-  form: {url: ''},
+  form: {url: '', uuid: ''},
```

Comment:
This change was once required to make my tests working, but I think it is no longer necessary. Kept it as it could be useful in the future.